source/_posts/2020/2020-01-27-switch-travis-to-github-actions-to-reduce-stress.md

```diff
@@ -226,7 +247,7 @@ How to work around this? If we create a new organization that is **open-source o
 
 **In Travis**, you could add a badge for the whole build on a specific branch. It was nice that it skipped allowed failure.
 
-**In GitHub Actions**, it's different. ~~The badge covers **1 specific workflow**. This workflow should contain all the relevant jobs~~.
+**In GitHub Actions**, it's different. [~~The badge covers **1 specific workflow**.](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository) This workflow should contain all the relevant jobs~~.
```

Comment:
I tried this before, but it forced me to couple dotzens of actions to one big noodle.

Badge is already above the repository covering full scope of all CI actions: https://github.com/rectorphp/rector/commit/468abb5c2c17675ade83448da98272530c7b1045

Badges are obsolete now :)

Reply:
How did you obtain the "full scope badge"? I cannot find any documentation and there is none in my repository. :)

Reply:
I'd try https://github.com/TomasVotruba/tomasvotruba.com/settings/actions

Reply:
![image](https://user-images.githubusercontent.com/924196/73267681-cf4f2280-41d9-11ea-865f-fc108d484ada.png)